pkg/export/attributes/attr_defs.go

```diff
@@ -99,6 +99,7 @@ func getDefinitions(groups AttrGroups) map[Section]AttrReportGroup {
 		Attributes: map[attr.Name]Default{
 			attr.K8sNamespaceName:   true,
 			attr.K8sPodName:         true,
+			attr.K8sContainerName:   true,
```

Comment:
Is container name low cardinality, it also makes me think we probably shouldn't make pod name on by default?

Reply:
I think it's low cardinality because it is not actually an ID but the name of the container that is given in the YAML file by the user. This means that all the pod instances would share the same set of container names (typically 1 or few)

In our helm chart, the pod attribute is disabled by default, along with many others. Maybe we could match this definitions files with what we already have in Helm.

Reply:
> In our helm chart, the pod attribute is disabled by default, along with many others. Maybe we could match this definitions' file with what we already have in Helm.

where? I can't find that

Reply:
Here: https://github.com/grafana/beyla/blob/main/charts/beyla/values.yaml#L211-L215 but I realize that the limitations only apply to network metrics